README.md

```diff
@@ -115,238 +115,165 @@ const jsonSchema = fromIntrospectionQuery(introspection, options);
-          required: []
-        },
-        completed: {
-          type: 'object',
-          properties: {
-            return: { '$ref': '#/definitions/Boolean' },
-            arguments: { type: 'object', properties: {}, required: [] }
-          },
-          required: []
-        },
-        color: {
-          type: 'object',
-          properties: {
-            return: { '$ref': '#/definitions/Color' },
-            arguments: { type: 'object', properties: {}, required: [] }
-          },
-          required: []
-        },
-        colors: {
-          description: 'A field that requires an argument',
-          type: 'object',
-          properties: {
-            return: { type: 'array', items: { '$ref': '#/definitions/Color' } },
-            arguments: {
-              type: 'object',
-              properties: {
-                filter: {
-                  type: 'array',
-                  items: { '$ref': '#/definitions/Color' }
-                }
-              },
-              required: [ 'filter' ]
-            }
-          },
-          required: []
-        }
-      },
-      required: [ 'id', 'name', 'colors' ]
-    },
-    SimpleTodo: {
-      type: 'object',
-      properties: {
-        id: {
-          type: 'object',
-          properties: {
-            return: { '$ref': '#/definitions/String' },
-            arguments: { type: 'object', properties: {}, required: [] }
-          },
-          required: []
-        },
-        name: {
-          type: 'object',
-          properties: {
-            return: { '$ref': '#/definitions/String' },
-            arguments: { type: 'object', properties: {}, required: [] }
-          },
-          required: []
-        }
-      },
-      required: [ 'id', 'name' ]
-    },
-    TodoUnion: {
-      oneOf: [
-        { '$ref': '#/definitions/Todo' },
-        { '$ref': '#/definitions/SimpleTodo' }
-      ]
-    },
-    TodoInputType: {
-      type: 'object',
-      properties: {
-        name: { '$ref': '#/definitions/String' },
-        completed: { '$ref': '#/definitions/Boolean' },
-        color: { '$ref': '#/definitions/Color', default: 'RED' }
-      },
-      required: [ 'name' ]
-    },
-    Color: {
-      type: 'string',
-      anyOf: [
-        {
-          description: 'Red color',
-          enum: [ 'RED' ],
-          title: 'Red color'
-        },
-        {
-          description: 'Green color',
-          enum: [ 'GREEN' ],
-          title: 'Green color'
-        }
-      ]
-    },
-    String: {
-      description: 'The `String` scalar type represents textual data, represented as UTF-8 character sequences. The String type is most often used by GraphQL to represent free-form human-readable text.',
-      type: 'string',
-      title: 'String'
-    },
-    Boolean: {
-      description: 'The `Boolean` scalar type represents `true` or `false`.',
-      type: 'boolean',
-      title: 'Boolean'
-    }
-  }
-}
+{ '$schema': 'http://json-schema.org/draft-06/schema#',
```

Comment:
@newhouse why this formatting change?

Is it prettier related?

Reply:
Not sure what happened there...fixed!